docs/source/deployment/databricks.md

```diff
@@ -160,7 +160,7 @@ Congratulations, you are now ready to run your Kedro project from the Databricks
 
 [Create your Databricks notebook](https://docs.databricks.com/notebooks/notebooks-manage.html#create-a-notebook) and remember to [attach it to the cluster](https://docs.databricks.com/notebooks/notebooks-manage.html#attach) you have just configured.
 
-In your newly created notebook put each code snippet from below into a separate cell and then [run all notebook cells](https://docs.databricks.com/notebooks/notebooks-use.html#run-all-cells):
+In your newly-created notebook, put each of the below code snippets into a separate cell, then [run all cells](https://docs.databricks.com/notebooks/notebooks-use.html#run-notebooks):
```

Comment:
```suggestion
In your newly-created notebook, put each of the below code snippets into a separate cell, and then [run all cells](https://docs.databricks.com/notebooks/notebooks-use.html#run-notebooks):
```
Also possible. See https://prowritingaid.com/grammar/1008094/Is-it-grammatically-correct-to-put-a-comma-before-then-